### PR TITLE
Feature branch: Portfolio Setup and Mobile Version Skeleton

### PR DESCRIPTION
**Pull request by abenezer-addisu**
In This Branch i did the following:

- Add HTML and CSS file 
- Mobile First Responsive 
- Add Linters file 
- Edit Readme file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;400;500;700;800&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="CSS/style.css">
+    <script src="https://kit.fontawesome.com/2b95109ad7.js" crossorigin="anonymous"></script>
+    <title>Abenezer Addisu</title>
+</head>
+<body>
+    <div class="home-page">
```
Comment (binhussen) [inline index.html]: - Great job for using semantic tags, I think the `div` tag is not usable to wrap the header and section, the `div` used only for unknown elements, so please remove it for better sematic reasons :+1:

File: CSS/style.css [hunk 2]
```diff
@@ -0,0 +1,414 @@
...
+  padding-right: 0;
+}
+
+.home h1 {
+  font-size: 40px;
+  margin-bottom: 0;
+  font-weight: 900;
+  line-height: 52px;
+  font-family: 'Poppins', sans-serif;
+}
```
Comment (binhussen) [inline CSS/style.css]: - Kindly, Use the Exact margins (top, bottom, left, right) and colors `#172B4D` defined for the template to have better design :+1: 

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|||


😎️

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @abenezer-addisu  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by frank1738 — COMMENTED** [review]
Hi @abenezer-addisu , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

-  [ ] You have submitted a project with failing linter checks. Kindly fix them and resubmit your project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by frank1738 — APPROVED** [review]
Hi @abenezer-addisu , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ Good documentation

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**